src/test/java/core/basesyntax/FileReaderImplTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.impl.FileReaderImpl;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class FileReaderImplTest {
```

Comment:
Let's check, that FileReader read not only the first (header) line, but all file content